applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -191,11 +193,12 @@ static void stream_started_cb(struct bt_bap_stream *stream)
 
 static void stream_stopped_cb(struct bt_bap_stream *stream, uint8_t reason)
 {
+	stream_stopped_reason = reason;
 
 	switch (reason) {
 	case BT_HCI_ERR_LOCALHOST_TERM_CONN:
 		LOG_INF("Stream stopped by user");
-		le_audio_event_publish(LE_AUDIO_EVT_NOT_STREAMING);
+		le_audio_event_publish(LE_AUDIO_EVT_SYNC_LOST);
```

Comment:
Why is LE_AUDIO_EVT_SYNC_LOST sent when the reason actually is BT_HCI_ERR_LOCALHOST_TERM_CONN ?

Reply:
The reason is that:
1. No matter if the reason is LOCALHOST_TERM_CONN or TIMEOUT, the sink needs to delete PA sync for making sure the sink has the ability to sync again
2. Sending LE_AUDIO_EVT_SYNC_LOST will trigger the "PA sync delete"
https://github.com/nrfconnect/sdk-nrf/blob/a29db7cfe09ab59edcf4e92517006a756c252ae7/applications/nrf5340_audio/src/audio/streamctrl_broadcast_sink.c#L257